examples/tutorial/vertical_federated_learning/cifar10-splitnn/cifar10_split_data_vertical.py

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import sys
+
+from splitnn.cifar10_vertical_data_splitter import Cifar10VerticalDataSplitter
+
+from nvflare.apis.fl_context import FLContext
+
+logging.basicConfig(stream=sys.stdout, level=logging.DEBUG)
+import argparse
+
+from nvflare.apis.fl_constant import ReservedKey
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--split_dir", type=str, default="/tmp/cifar10_vert_splits", help="output folder")
+    parser.add_argument("--overlap", type=int, default=10_000, help="number of overlapping samples")
+    args = parser.parse_args()
+
+    splitter = Cifar10VerticalDataSplitter(split_dir=args.split_dir, overlap=args.overlap)
+
+    # set up a dummy context for logging
```

Comment:
This way, it can also be used as an FL component similar to the cifar10 FedAvg [example](https://github.com/NVIDIA/NVFlare/blob/3111a23a95d645ac17afb462a04f722997535210/examples/cifar10/cifar10-sim/job_configs/cifar10_fedavg/cifar10_fedavg/config/config_fed_server.json#L15) for simulation pruposes.

Reply:
For example, you can automate the runs by adding the `Cifar10VerticalDataSplitter` to your job config and run experiments with different overlaps in parallel.